README.md

```diff
@@ -150,3 +150,19 @@ Thanks also go to the developers of the [msvc-dev-cmd](https://github.com/ilammy
     <img valign="middle" height="100" src=".graphics/sbml-logo-only.png">
   </a>
 </div>
+
+Research notice
```

Comment:
since this is markdown, could you please give it the same markdown heading as the one we are using in the document?